docs/source/ultimate.rst

```diff
@@ -24,7 +24,7 @@ We will keep the private portion (name, contact information etc.) of your data i
 
 **Letting us use and share your data is voluntary.  However, you must be willing to share your data in this way in order to participate in this study. If you are not willing, you cannot participate in this study.**
 
-By signing below, you agree to provide your data for future research. You agree that these may be shared with other investigators at other institutions from around the world. The details, results, and implications of these studies are unknown.
+By signing below, you agree to provide your data for future research. You agree that these may be shared with other investigators at other institutions from around the world as well as the general public. The details, results, and implications of these studies are unknown.
```

Comment:
shouldn't it then be in both versions?

Reply:
The other version has two levels of access so not all data will be available to general public.

Reply:
yeah, I have missed that 2nd version was talking about "authorized investigators".
BUT it is the "two access types version", where some data **is** shared publicly, so it would only be logical to extend that sentence in the second version with similar ending, e.g.

_with other authorized investigators at other institutions from around the world, **and some data will be shared with the general public**._

Reply:
don't you think @chrisfilo so? ;)

Reply:
In general I agree, but I am more concerned about the workflow for updating all of the translations along with the English version. We need to come up with something.

Reply:
Yeap, the same here. Someone should check how it is done in other text documents oriented projects. I know how it is done typically for shorter ones in software projects where each sentence in English is pretty much a key for which translations are then assigned.

Reply:
Also would help to separate them all into different documents asap

Reply:
ref: http://www.sphinx-doc.org/en/stable/intl.html   yet to try

Reply:
Ok, translations workflow is somewhat streamlined (separate files, versioning), so it might indeed be good to add these explicit statements for now at least into English version.